helix-term/src/ui/editor.rs

```diff
@@ -127,6 +127,24 @@ impl EditorView {
         Self::render_text_highlights(doc, view.offset, inner, surface, theme, highlights);
         Self::render_gutter(editor, doc, view, view.area, surface, theme, is_focused);
 
+        // Render color columns
+        editor
+            .config()
+            .color_column
+            .iter()
+            .filter_map(|column| {
+                column
+                    .checked_sub(1 + view.offset.col as u16)
+                    .map(|x| {
+                        inner
+                            .with_height((view.last_line(doc) - view.offset.row + 1) as u16)
+                            .clip_left(x)
+                            .with_width(1)
+                    })
+                    .filter(|area| area.left() < inner.right())
+            })
+            .for_each(|area| surface.set_style(area, theme.get("ui.colorcolumn")));
```

Comment:
I think we can probably default this to `red`.

Reply:
There's already `ui.highlight` that can be used.

Reply:
I'd scope this as `ui.highlight.ruler` so that it can be styled independently but have a fallback

Reply:
This is so neat! This dot notation fallback system is very powerful.
I allowed myself to add `ui.highlight` along with the new `ui.highlight.ruler` in the list of available theme keys as I think it was missing.